src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -8,13 +8,23 @@
 
```

Comment:
Ideally, this file wouldn't have to be modified every time we have a new version. I created [#AB72048](https://microsofthealth.visualstudio.com/Health/_workitems/edit/72048) to look into this.